app/models/participant_outcome_api_request.rb

```diff
@@ -0,0 +1,10 @@
+class ParticipantOutcomeAPIRequest < ApplicationRecord
+  belongs_to :participant_outcome
+
+  validates :ecf_id,
+            presence: { message: "Enter an ECF ID" },
```

Comment:
Can we move these to the translations file

Reply:
Hi @ethax-ross 👍 tks.. ended up removing the custom message as don't think it's needed in this case.